charts/service/templates/service.yaml

```diff
@@ -3,13 +3,19 @@ kind: Service
 metadata:
   name: {{ include "service.fullname" . }}-service
   labels:
-  {{- include "service.labels" . | nindent 4 }}
+    {{- include "service.labels" . | nindent 4 }}
 spec:
   type: {{ .Values.service.type }}
   ports:
     - port: {{ .Values.service.port }}
       targetPort: {{ .Values.service.targetPort }}
       protocol: TCP
       name: http
+    {{- if .Values.extraService }}
+    - port: {{ .Values.extraService.Port }}
+      targetPort: {{ .Values.extraService.targetPort }}
+      protocol: TCP
+      name: metrics
```

Comment:
this should also be a named property

Reply:
i named it extraService to be a general name not specified for a specific function

Reply:
is this what you meant?

Reply:
```
 name: metrics
```